TheMovieDb/Controller/ViewController.swift

```diff
@@ -77,5 +77,15 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         return cell
     }
     
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if let vcMovieSelected = storyboard?.instantiateViewController(identifier: "MovieInfo") as? MovieInfoViewController {
+            vcMovieSelected.movie = resultsMovie[indexPath.row]
+            navigationController?.pushViewController(vcMovieSelected, animated: true)
+            print("You selected cell number \(indexPath.row)")
+        }
+        
+        
```

Comment:
1. Delete the extra space.
2. Delete the `print("You selected cell number \(indexPath.row)")`.
3. The identifier "MovieInfo" should be in a constant.
4. Be more descriptive vcMovieSelected.